js/core/renderer.d.ts

```diff
@@ -23,7 +23,7 @@ export interface dxElementWrapper {
 
   clone(): this;
 
-  closest(selector: string): this;
+  closest(selector: string | dxElementWrapper): this;
```

Comment:
Question.

Can we don't make this change in this PR?

Because if we touch any ```*.d.ts``` we need additional approve from ApiReviewers